docker/Makefile

```diff
@@ -125,9 +125,18 @@ test-env-setup-with-no-build-ccf:
 	-$(DOCKER_COMPOSE_COMMAND) $(DOCKER_COMPOSE_OPTS_CCF) down
 	# - start services
 	$(DOCKER_COMPOSE_COMMAND) $(DOCKER_COMPOSE_OPTS_CCF) up -d
+
+	# Before registration, it must be ensured that CCF is up and configured.
+	# Otherwise, the registration can encounter two possible scenarios:
+	# 1) CCF is not up, so the registration times out
+	# 2) CCF is up but not configured, so the registration fails with "not a known member"
+	# A wait of 10 seconds appears to be enough.
+	@echo "Waiting a few seconds for CCF to be up and configured..."
+	sleep 10
+
```

Comment:
in other places we have specifically set up a loop with a test for functional service. the pure sleeps seems pretty adhoc. Is there a ping operation we can use (add?) in order to test that the ledger is functioning?

Reply:
> the pure sleeps seems pretty adhoc

True

>  Is there a ping operation we can use (add?) in order to test that the ledger is functioning?

The risk is that you can still incur in scenario (2): ledger functioning but not configured.
@prakashngit , any thoughts on this?